api/observation.go

```diff
@@ -182,9 +182,10 @@ func extractQueryParameters(urlQuery url.Values, validDimensions []string) (map[
 	for dimension, option := range urlQuery {
 		queryParamExists := false
 		for _, validDimension := range validDimensions {
-			if dimension == validDimension {
+			// Ignore case sensitivity
+			if strings.ToLower(dimension) == strings.ToLower(validDimension) {
 				queryParamExists = true
-				queryParameters[dimension] = option[0]
+				queryParameters[validDimension] = option[0]
```

Comment:
Should we be using the lower cased version as the key here?

Reply:
It's just to assist the user, if they typed `......./observations?Geography=K00002` instead of `......./observations?geography=K00002`, it will still successfully find the observations. It might make for easier reading if it was like so:

```
queryParameters[strings.ToLower(dimension)] = option[0]
```